### ISSUE TITLE
In columns mode, if a item has no subtitle the lowercase letters of ygjqp gets cut off

### ISSUE THREAD

**Issue by No0Vad** [bastienwirtz/homer#794]
**Describe the bug**
In columns mode, if a item has no subtitle letters with a "tail" (unsure what they are called, but ygjqp) gets cut off when hovering above them.

Can be seen on the [demo](https://homer-demo.netlify.app/) too if you manually add `no-icon` to `div.media` and remove the `p.subtitle` node. That's where my images are from.

**Expected behavior**
Them to not get cut off

**Logs & errors**
None

**Screenshots**
Hover, look at the `g`.
![image](https://github.com/bastienwirtz/homer/assets/9308455/08970e7d-0915-4ac4-ad23-46ae2c080908)

Normal
![image](https://github.com/bastienwirtz/homer/assets/9308455/cdcfa644-8f09-4f64-8164-8dc62a8b3619)

**Configuration**
Basically any item with a text containing ygjq or p in lowercase, just with no subtitle.

**Comment by bastienwirtz** [comment]
Hello, 

Thanks for reporting this, it has been fixed 👍
